es-vue-base/src/lib-components/EsSlider.vue

```diff
@@ -17,15 +17,15 @@
             <!-- Tooltip above the slider thumb -->
             <template #tooltip="{ value, focus }">
                 <div
-                    class="slider-tooltip d-flex h5 align-items-center text-white justify-content-center m-0 bg-primary"
+                    class="slider-tooltip d-flex h4 align-items-center text-white justify-content-center m-0 bg-blue-600 font-weight-boldest"
                     :class="[{ focus }]">
                     {{ tooltipFormatter(value) }}
                 </div>
             </template>
             <!-- Labels below the process bar -->
             <template #label="{ label, active }">
                 <div :class="['slider-label', 'vue-slider-mark-label', { active }]">
-                    {{ label }}
+                    ${{ label }}
```

Comment:
This slider may be used in cases that don't involve dollar amounts, so I'm not sure we should hardcode a "$" in front of each label - does the library provide any label formatter option, similar to what we're doing with tooltip formatter?